### PR TITLE
add H3C to the list of ADOPTERS 

### PR DESCRIPTION
**Pull request by zxb789**
add Kubevirt as part of  CloudOS  in H3C

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -5,4 +5,5 @@ This is a likely incomplete list of KubeVirt adopters - end-users and distributo
 | User | Since | Use-Case |
 |-|-|:-|
 | Red Hat | 2016 | As a distributor we are building OpenShift Virtualization on KubeVirt in order to enable VM workloads and -flows on Kubernetes. |
-| [Kubermatic](https://kubermatic.com) | 2019 | As a distributor we are running KubeVirt to enable VM workload on Kubermatic Virtualization. Follow the [link](https://www.kubermatic.com/products/kubevirt/) for more information. |
\ No newline at end of file
+| [Kubermatic](https://kubermatic.com) | 2019 | As a distributor we are running KubeVirt to enable VM workload on Kubermatic Virtualization. Follow the [link](https://www.kubermatic.com/products/kubevirt/) for more information. |
+| H3C | 2019 | As a distributor we are running KubeVirt in order to enable VM workloads on Kubernetes and distribute to more and more customer . Follow the  [link](https://www.h3c.com/en/Products_Technology/Enterprise_Products/Cloud_Computing/Cloud_Computing_Products/H3C_CloudOS/H3C_CloudOS_full-stack/) for more product information |
```
Comment (dankenigsberg) [inline ADOPTERS.md:9]: I think that the former rows are written in cumbersome language. Would you consider writing something like

We distribute KubeVirt as part of [CloudOS](https://www.h3c.com/en/Products_Technology/Enterprise_Products/Cloud_Computing/Cloud_Computing_Products/H3C_CloudOS/H3C_CloudOS_full-stack/)  to enable VM workloads on Kubernetes at customer sites 

I find it simpler shorter and easier to understand.

**Review by dankenigsberg — CHANGES_REQUESTED** [review]

**Comment by dankenigsberg** [comment]
> add use case as pr

You should use the pr message to explain this pr in more detail. Eg you can tell more about h3c and its adoption of KubeVirt.

File: ADOPTERS.md (hunk 1 above)
Comment (zxb789) [reply ADOPTERS.md:9]: ok,great,let me modify it

**Comment by zxb789** [comment]
> > add use case as pr
> 
> You should use the pr message to explain this pr in more detail. Eg you can tell more about h3c and its adoption of KubeVirt.

File: ADOPTERS.md (hunk 1 above)
Comment (zxb789) [reply ADOPTERS.md]: ok that  haved been  modified

**Comment by rmohr** [comment]
/approve

@zxb789 thanks  for the PR. Our CI system will take care of the merge. It may need a few attempts but you don't have to follow up.

**Pull request merged**